Sourcery/ConfigOptions.md.stencil

```diff
@@ -41,7 +41,7 @@
 ## {{ type.localName|lowercase }}.{{ variable.name }}
 {% endif %}
 
-**Type:** {{ variable.typeName|replace:"?","" }}<br />
+**Type:** {{ variable.typeName|replace:"?",""|replace:"EntityExclude","String" }}<br />
```

Comment:
I wouldn't worry about doing this, as long as the documentation is good and has examples. 

If we want to make the 'type' accurate for yaml/json then we can do it universally

Reply:
I strongly recommend we only present only yaml acceptable values. `EntityExclude` is entirely for our development purposes and won't make sense to the user reading documentation even with examples.

Reply:
Reading up on Sourcery annotations. I might be able to make a type replacement annotation.

Reply:
I'll drop this point for the implementation of this feature. However, I still propose we make surfacing documentation types their yaml types instead of our underlying types.

Reply:
We could add a custom `'defaultValue'` annotation to use instead but it might not be worth worrying about this too much right now, I'm trying out yet another option for handling documentation although I'm not too sure if it will work just yet or not 😅